tests/integration/sync/syncModel.test.ts

```diff
@@ -89,20 +85,15 @@ describe.concurrent("Sync environment from folder", () => {
       .then(stats => stats.isDirectory())
       .catch(() => false);
 
-    const typesJsonExists = await fsPromises.stat(`${folderPath}/${contentTypesFileName}`)
-      .then(stats => stats.isFile())
-      .catch(() => false);
-    const snippetJsonExists = await fsPromises.stat(`${folderPath}/${contentTypeSnippetsFileName}`)
-      .then(stats => stats.isFile())
-      .catch(() => false);
-    const taxonomiesJsonExists = await fsPromises.stat(`${folderPath}/${taxonomiesFileName}`)
-      .then(stats => stats.isFile())
-      .catch(() => false);
-
     expect(folderExists).toEqual(true);
-    expect(typesJsonExists).toEqual(true);
-    expect(snippetJsonExists).toEqual(true);
-    expect(taxonomiesJsonExists).toEqual(true);
+
+    const result = await Promise.all(
+      Object.entries(fileNames).map(([, filename]) =>
+        fsPromises.stat(`${folderPath}/${filename}`).then(stats => stats.isFile()).catch(() => false)
+      ),
+    );
+
+    expect(result.every(res => res)).toEqual(true);
```

Comment:
Thank you for the improvement. This will always check that all the files are present. But I suggest we also improve the error message in case it fails. We can create an object with keys set to fileNames and values would be `true`/`false` depending on the file's existence. Then, we can do `expect(filesExistence).toStrictEqual(allFilesExist)`. Then the error message would tell you precisely which files are missing.